src/core/utils/MultisigGraphUtils.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MultisigAccountGraphInfo } from '../../model/account/MultisigAccountGraphInfo';
+import { MultisigAccountInfo } from '../../model/account/MultisigAccountInfo';
+
+/**
+ * Hash utilities for SecretLock hashing
+ */
+export class MultisigGraphUtils {
+    public static parseObjectProperties(obj, parse): any {
+        for (const k in obj) {
+            if (typeof obj[k] === 'object' && obj[k] !== null) {
+                this.parseObjectProperties(obj[k], parse);
+            } else if (Object.prototype.hasOwnProperty.call(obj, k)) {
+                parse(k, obj[k]);
+            }
+        }
+    }
+    public static getMultisigChildren(
+        multisigAccountGraphInfoMapped: MultisigAccountInfo[][],
+    ): Array<{ address: string; children: string[] }> {
```

Comment:
why array<T>, can a model class/interface be used here?

Reply:
@rg11 I didn't think that we need to use a new model/type to handle such a case, do we?

Reply:
`Array<{ address: string; children: string[] }>` is type friendly, probably fit the wallet's purpose, but would be nice to create a type / interface for it